gradio/buttons.py

```diff
@@ -0,0 +1,61 @@
+""" Predefined buttons with bound events that can be included in a gr.Blocks for convenience. """
+
+from typing_extensions import Literal
+
+import json
+from gradio_client.documentation import document, set_documentation_group
+
+from gradio.components import Button, Component
+
+set_documentation_group("component")
```

Comment:
Added to the component section for now. Once we have more buttons, we can have a dedicated section for Buttons in the docs page.